codespell_lib/_codespell.py

```diff
@@ -694,6 +694,7 @@ def ask_for_word_fix(
         fixword = fix_case(wrongword, misspelling.data)
         while not r:
             print(f"{line}\t{wrongword} ==> {fixword} (Y/n) ", end="")
+            sys.stdout.flush()
```

Comment:
More compact
```suggestion
            print(f"{line}\t{wrongword} ==> {fixword} (Y/n) ", end="", flush=True)
```

Reply:
Even better indeed, thanks!
I guess we could also use it to replace the line 720?

Reply:
Sure, feel free